recipes/altair-aitools-devkit/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "altair-aitools-devkit" %}
+{% set version = "1.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://maven.rapidminer.com/repository/pypi-snapshot/packages/altair-aitools-devkit/{{ version }}/altair_aitools_devkit-{{ version }}.tar.gz
+  sha256: b496405744ff6b0954e717ba6cc50c0558aa582a15c29e3af5e940325758056b
+
+build:
+  entry_points:
+    - build-extension = altair_aitools.ext._builder:build_extension
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
```

Should be the same as `run`.

```suggestion
    - python >=3.8.0,<4.0.0
```